barracuda_secure_edge/assets/logs/barracuda_secure_edge.yaml

```
@@ -0,0 +1,495 @@
+      - Barracuda SecureEdge
+    name: eventname
+    path: secure_edge.eventname
+    source: log
+    type: string
+  - description: "Level of the secure edge event"
+    facetType: list
+    groups:
+      - Barracuda SecureEdge
+    name: level
+    path: secure_edge.level
+    source: log
+    type: string
+  - description: "Module of the secure edge event"
+    facetType: list
+    groups:
+      - Barracuda SecureEdge
+    name: module
+    path: secure_edge.module
+    source: log
+    type: string
+  - description: "Product name of the secure edge event"
+    facetType: list
+    groups:
+      - Barracuda SecureEdge
+    name: productname
+    path: secure_edge.productname
+    source: log
+    type: string
+  - description: "Protocol of the secure edge event"
+    facetType: list
+    groups:
+      - Barracuda SecureEdge
+    name: protocol
+    path: secure_edge.protocol
+    source: log
+    type: string
+  - description: "Rule of the secure edge event"
+    facetType: list
+    groups:
+      - Barracuda SecureEdge
+    name: rule
+    path: secure_edge.rule
+    source: log
+    type: string
+  - description: "Source IP address of the secure edge event"
+    facetType: list
+    groups:
+      - Barracuda SecureEdge
+    name: src_ip
+    path: secure_edge.src_ip
+    source: log
+    type: string
+pipeline:
+  type: pipeline
+  name: Barracuda_Secure_Edge
+  enabled: true
+  filter:
+    query: source:barracuda_secure_edge
+  processors:
+    - type: pipeline
+      name: Box Auth Access
+      enabled: true
+      filter:
+        query: source:barracuda_secure_edge service:auth
+      processors:
+        - type: grok-parser
+          name: Box Auth Access
+          enabled: true
+          source: message
+          samples:
+            - "2025 04 02 12:20:39 +00:00 Notice       root : TTY=pts/0 ;
+              PWD=/etc/yum.repos.d ; USER=root ; COMMAND=/bin/systemctl rsyslog
+              status"
+            - "2025 04 02 12:20:39 +00:00 Info     : pam_unix(sudo:session):
+              session opened for user root by root(uid=0)"
+            - "2025 03 31 11:46:17 +00:00 Info     LOGIN ATTEMPT: control[5405]:
+              Login localhost_hapair_daemon from 127.0.0.1 : Allowed."
+            - "2025 03 31 12:16:35 +00:00 Info     boxconfig[26379]: Session
+              localhost_CloudGen-WAN: Closed"
+            - "2025 05 27 19:47:27 +00:00 Info     : pam_unix(sudo:session):
+              Login root from 10.10.16.1: Denied: Invalid password"
+          grok:
+            supportRules: ""
+            matchRules: >-
+              autoFilledRule3 %{date("yyyy MM dd HH:mm:ss Z"):timestamp}\s*%{word:level}\s*%{word:module}\s*:\s*(?<message>.+)
+              autoFilledRule1 %{date("yyyy MM dd HH:mm:ss Z"):timestamp}\s*%{word:level}\s*%{data:action}:\s*%{word:type}\[%{number:type_id}\]:\s*(?<message>.+)
+              autoFilledRule4 %{date("yyyy MM dd HH:mm:ss Z"):timestamp}\s*%{word:level}\s*(:\s+)?(?<message>.+)
```

```suggestion
              autoFilledRule3 %{date("yyyy MM dd HH:mm:ss Z"):timestamp}\s*%{word:level}\s*%{word:module}\s*:\s*%{greedyData:message}
              autoFilledRule1 %{date("yyyy MM dd HH:mm:ss Z"):timestamp}\s*%{word:level}\s*%{data:action}:\s*%{word:type}\[%{number:type_id}\]:\s*%{greedyData:message}
              autoFilledRule4 %{date("yyyy MM dd HH:mm:ss Z"):timestamp}\s*%{word:level}\s*(:\s+)?%{greedyData:message}
```

Please make this change, it should be equivalent and grok syntax should be used.